Makefile.core.mk

```diff
@@ -148,16 +148,16 @@ export
 test: envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -coverprofile cover.out
 
-.PHONY: test.scorecard ## Runs the operator scorecard test.
-test.scorecard: operator-sdk
+.PHONY: test.scorecard
+test.scorecard: operator-sdk ## Run the operator scorecard test against an existing cluster.
```

Comment:
Keep the original help. We no longer need an existing cluster. The test deploys a kind cluster.

Reply:
done